### DISCUSSION TITLE
Can't accept invitation for organisation to join enterprise

### DISCUSSION THREAD

**Discussion by dkfellows** [community/community#145215] [Enterprise]
I'm in the middle of setting up an organisation that will be within my enterprise, and when I go to the enterprise page I can see that the organisation has been invited, but I can't find how to accept the invitation! The billing settings page (as linked to from the enterprise page) simply has no link or button for me to complete accepting the invitation.

Am I missing something obvious here?

**Comment by queenofcorgis** [comment]
Hi there @dkfellows 👋🏼 

The organization's owner should have received an email invite to accept the invitation. [Linking this doc](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-accounts-and-repositories/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise#inviting-an-existing-organization) that details the steps of inviting an existing organization.
